ecs-init/volumes/efs_driver.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package volumes
```

Comment:
consider getting rid of the efs* descriptions and leaving this as a generic mount helper for now. As written, this could be reused for any mount helper installed on the host.

MB we have this be the default case, and then add EFS specific logic to a different struct via embedding. Just a thought.

Reply:
called it generically as "ECS Volume Driver" which has only EFS mount helper as of now. we can make it an interface in the future to have different mount helpers :)